test/test_other.py

```diff
@@ -1689,6 +1689,18 @@ def test_stdin(self, args):
         os.system(f'cat in.txt | {cmd} > out.txt')
       self.assertContained('abcdef\nghijkl\neof', read_file('out.txt'))
 
+  @also_with_noderawfs
+  @crossplatform
+  def test_module_stdin(self):
+    self.set_setting('FORCE_FILESYSTEM')
+    self.set_setting('EXIT_RUNTIME')
+    create_file('pre.js', '''
+const data = 'hello, world!\\n'.split('').map(c => c.charCodeAt(0));
+Module['stdin'] = () => data.shift() || null;
+''')
+    self.emcc_args += ['--pre-js', 'pre.js']
+    self.do_runf(test_file('module/test_stdin.c'), 'hello, world!')
```

Comment:
No need for `test_file` here.

Reply:
Ah, great, done with commit c39e5b14d3510f4fa673feeec3c76a85664a645f.